tests/ci/kickoff_devicefarm_job.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -exuo pipefail
+
+# -e: Exit on any failure
+# -x: Print the command before running
+# -u: Any variable that is not set will cause an error if used
+# -o pipefail: Makes sure to exit a pipeline with a non-zero error code if any command in the pipeline exists with a
+#              non-zero error code.
+
+###########################
+# Main and related helper #
+###########################
+
+function script_helper() {
+  cat <<EOF
+This script helps compile AWSLCAndroidTestRunner and kick off the device farm python script with the arguments needed.
+
+Options:
+    --help                          Displays this help menu
+    --test-name						          Name of current test.
+    --devicefarm-project-arn        The devicefarm project's arn. Default to team account's.
+    --devicefarm-device-pool-arn    The device pool's arn.
+    --fips                          If we're compiling for FIPS or not. Will always compile for the FIPS, shared, release build.
+    --release                       If we're compiling for Release or Debug.
+    --shared                        If we're compiling for Shared or Static.
+    --action                        Required. The value can be:
+	                                   'start-job': kicks off a device farm job.
+EOF
+}
+
+function export_global_variables() {
+  # If these variables are not set or empty, defaults are exported, but the |main-apk| and |test-apk| must be set.
+  if [[ -z "${ANDROID_TEST_NAME+x}" || -z "${ANDROID_TEST_NAME}" ]]; then
+    export ANDROID_TEST_NAME='AWS-LC Android Test'
+  fi
+  if [[ -z "${DEVICEFARM_PROJECT+x}" || -z "${DEVICEFARM_PROJECT}" ]]; then
+    export DEVICEFARM_PROJECT='arn:aws:devicefarm:us-west-2:620771051181:project:d1e78543-a776-49c5-9452-9a2b3448b728'
```

Comment:
Maybe the arn should be defined above so it's easier to change if we need to change the default

Reply:
Good idea